extensions/campeonato-brasileiro/package.json

```
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "campeonato-brasileiro",
+  "title": "Campeonato brasileiro",
```

```suggestion
  "title": "Campeonato Brasileiro",
```
I'm adding a code suggestion to the extension title for using Title Cased. [Here](https://developers.raycast.com/basics/prepare-an-extension-for-store#extensions-and-commands-naming) you can find more info.
After the eslint config file installation, you'd have found this by running `npm run lint`